src/java.net.http/share/classes/jdk/internal/net/http/common/DebugLogger.java

```diff
@@ -53,60 +115,65 @@ final class DebugLogger implements Logger {
     private final boolean traceOn;
 
     /**
      * Create a logger for debug traces.The logger should only be used
      * with levels whose severity is {@code <= DEBUG}.
-     *
-     * By default, this logger will forward all messages logged to the supplied
-     * {@code logger}.
-     * But in addition, if the message severity level is {@code >=} to
-     * the provided {@code errLevel} it will print the messages on System.err,
-     * and if the message severity level is {@code >=} to
-     * the provided {@code outLevel} it will also print the messages on System.out.
+     * <p>
+     * By default, this logger will print message whose severity is
+     * above the severity configured in the logger {@code config}
+     * <ul>
+     *     <li>If {@code config.outLevel()} is not Level.OFF, messages
+     *     whose severity are at or above that severity will be directly
+     *     printed on System.out</li>
+     *     <li>If {@code config.errLevel()} is not Level.OFF, messages
+     *     whose severity are at or above that severity will be directly
+     *     printed on System.err</li>
+     *     <li>If {@code config.logLevel()} is not Level.OFF, messages
+     *     whose severity are at or above that severity will be forwarded
+     *     to the supplied {@code logger}.</li>
+     * </ul>
      * <p>
      * The logger will add some decoration to the printed message, in the form of
      * {@code <Level>:[<thread-name>] [<elapsed-time>] <dbgTag>: <formatted message>}
      *
      * @apiNote To obtain a logger that will always print things on stderr in
      *          addition to forwarding to the internal logger, use
-     *          {@code new DebugLogger(logger, this::dbgTag, Level.OFF, Level.ALL);}.
+     *          {@code new DebugLogger(logger, this::dbgTag,
+     *                                 LoggerConfig.LOG.withErrLevel(Level.ALL));}.
      *          To obtain a logger that will only forward to the internal logger,
-     *          use {@code new DebugLogger(logger, this::dbgTag, Level.OFF, Level.OFF);}.
+     *          use {@code new DebugLogger(logger, this::dbgTag, LoggerConfig.LOG);}.
      *
      * @param logger The internal logger to which messages will be forwarded.
-     *               This should be either {@link #HPACK} or {@link #HTTP};
+     *               This should be either {@link #WS}, {@link #HPACK}, or {@link #HTTP};
      *
      * @param dbgTag A lambda that returns a string that identifies the caller
      *               (e.g: "SocketTube(3)", or "Http2Connection(SocketTube(3))")
-     * @param outLevel The level above which messages will be also printed on
-     *               System.out (in addition to being forwarded to the internal logger).
-     * @param errLevel The level above which messages will be also printed on
-     *               System.err (in addition to being forwarded to the internal logger).
+     * @param config The levels above which messages will be printed to the
```

Comment:
This should be `The levels at or above which ....`

Reply:
Done.